### PR TITLE
optimize getting started path

### PR DESCRIPTION
**Pull request by dckc**
- demote Documentation Guide - "how to read the documentation" should be implicit by following conventions; for now, just get it out of the way of folks getting started
 - format installation instructions conventionally (fenced code rather than table)
   - add quick-start
   - integrate Nov 2020 video
 - format "starting a project" steps conventionally
   - integrate Nov 2020 video

trial deployment:
https://ag-docs-org.netlify.app/documentation/getting-started/before-using-agoric.html
https://ag-docs-org.netlify.app/documentation/getting-started/start-a-project.html#initialize-demo-from-dapp-template

### CODE REVIEW

**Review by michaelfig — APPROVED** [review]

**Comment by dckc** [comment]
PTAL. Note the embedded video.

trial deployment:
https://ag-docs-org.netlify.app/documentation/getting-started/before-using-agoric.html
https://ag-docs-org.netlify.app/documentation/getting-started/start-a-project.html#initialize-demo-from-dapp-template

**Comment by dckc** [comment]
@kennyrowe please take a look at this

File: main/getting-started/start-a-project.md [hunk 1]
```diff
@@ -6,115 +6,104 @@ It has not yet been formally tested or hardened.
...
+This presentation includes starting a project, but note an outdated detail:
+
+ - In the REPL `x~.go()` tildot support has been postponed; use `E(x).go()`.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/w0By22jYhJA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+:::
+
+## Initialize `demo` from Dapp Template
+
+Use the [Aagoric CLI](/guides/agoric-cli/commands.md) to fetch from a Dapp template
```
Comment (michaelfig) [inline main/getting-started/start-a-project.md:30]:
```suggestion
Use the [Agoric CLI](/guides/agoric-cli/commands.md) to fetch from a Dapp template
```

**Review by michaelfig — APPROVED** [review]
LGTM!  Shipit!

**Pull request merged**